evaluation/run_eval.py

```diff
@@ -55,6 +55,7 @@ def get_config(
     config.set_llm_config(llm_config)
     agent_config = AgentConfig(
         enable_prompt_extensions=False,
+        enable_history_truncation=False,
```

Comment:
This is for both 1) backward compatibility, since our existing experiments didn't have this feature yet, and 2) empirically this doesn't work well for our settings.